modules/card/css/README.md

```diff
@@ -95,12 +95,33 @@ Card positioning can be changed by adding a modifier to the container class,
 ```html
 <div class="wdc-card-container wdc-card-container-space-around">
   <div class="wdc-card-2">
-    <div class="wdc-card-header">
+    <h3 class="wdc-card-header">
       Card Header
-    </div>
+    </h3>
     <div class="wdc-card-body">
       .wdc-card-container .wdc-card-container-space-around
     </div>
   </div>
 </div>
 ```
+
+#### Card Depth
+
+Five levels of depth available in CSS classes
```

Comment:
In case we ever change the number of depths available we shouldn't use the exact number. Can we change this to:

"Depth 2 is the default. You can use the [depth classes from @workday/canvas-kit-css-core](../../core/css/README.md#depth) to set a custom depth for your card."